profiles/common/modules/custom/nexteuropa_token/src/Entity/ViewModeTokenHandler.php

```diff
@@ -154,7 +154,8 @@ protected function parseToken($original, $item = 'entity_id') {
   public function getEntityViewModes($token_type) {
 
     $view_modes = array();
-    $entity_type = $token_type == 'term' ? 'taxonomy_term' : $token_type;
+    $token_types = token_get_entity_mapping('token');
```

Comment:
Function `token_get_entity_mapping()` by default provides mapping for the 'token' value;